.travis.yml

```diff
@@ -15,32 +15,24 @@ env:
     - secure: kCOLgEveEDprSL3fOBeCc1/nmHVYEcNh/lRRWiJywFehOzKBTBF40pAWX7JtsWUPG8aPKGYFEE4sjYd+DF2DRknbuB2ZfAEgs9Xp9h5Mx1KztJYNiCfEnLciBBRDXA7t1M3TLKyT3ptA+Kk307cH2QHv9sUUnhnf0IofJyQhQvA=
   
 jdk:
-  #- openjdk7
-  - openjdk8
+  - openjdk11
 
 before_install:
   # required for testing jdk 11
   - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh -O $TRAVIS_BUILD_DIR/install-jdk.sh
   - chmod u+x $TRAVIS_BUILD_DIR/install-jdk.sh
 
 install:
-  # compile and test java 8
+  # compile and test java 11
   - mvn -B -V clean test
```

Comment:
@magro: Why do we run the tests in the `install` phase? They are run again immediately in the `script` phase by `mvn -B install`. Imho the `install` phase can be removed.

Reply:
Not sure TBH, so I don't know a good reason why we should not get rid of the install stage.

Reply:
@magro: OK I get it now. The [default behavior](https://docs.travis-ci.com/user/languages/java/#projects-using-maven) of Travis is to run:

```
mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

It probably was necessary to override this behavior before but I don't see a reason for it now. I have further simplified the build config to be more aligned with Travis default behavior in https://github.com/EsotericSoftware/kryo/commit/8766e270d01760cf3a64efba26381a28ff54dae4 but you merged my PR seconds before I pushed the change.

If you want you can apply the changes manually. Currently, we run one more `install` than is really necessary.

Reply:
Can you submit a separate PR please?

Reply:
Sure! Here you go #728